src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -1,6 +1,8 @@
+import { PaymentStrategyType } from '../..';
```

Comment:
Can you add index to the path please, because it looks weird? 🙏

```suggestion
import { PaymentStrategyType } from '../../index';
```

Reply:
Fixed

Reply:
fixed